Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -38,14 +38,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if let storedApiKey = storage.apiKey {
             apiKey = storedApiKey
         }
-        CustomerIO.initialize(siteId: siteId, apiKey: apiKey, region: .US) { config in
-            config.logLevel = self.storage.isDebugModeEnabled ?? true ? .debug : .error
-            config.autoTrackDeviceAttributes = self.storage.isTrackDeviceAttrEnabled ?? true
-            config.backgroundQueueSecondsDelay = Double(self.storage.bgQDelay ?? "30") ?? 30
-            config.backgroundQueueMinNumberOfTasks = Int(self.storage.bgNumOfTasks ?? "10") ?? 10
-            config.autoTrackScreenViews = self.storage.isTrackScreenEnabled ?? true
+        let writeKey = "\(siteId):\(apiKey)"
+        CustomerIO.initialize(writeKey: writeKey) { sdkConfig, cdpConfig in
```

Comment:
we shouldn't be needing both `sdkConfig, cdpConfig`? old users come via sdkConfig and new ones should use cdpConfig?

Reply:
**Update:** We discussed this on call, and decided to remove `SdkConfig` for CDP users for now and update it from

```swift
static func initialize(
    writeKey: String,
    configure configureHandler: ((inout SdkConfig, inout DataPipelineConfigOptions) -> Void)?
)

CustomerIO.initialize(writeKey: writeKey) { sdkConfig, cdpConfig in
     sdkConfig.logLevel = .debug
     sdkConfig.autoTrackScreenViews = true

     cdpConfig.flushInterval = 30
     cdpConfig.flushAt = 10
}
```

to the following

```swift
static func initialize(
    writeKey: String,
    configure configureHandler: ((inout SdkConfig, inout DataPipelineConfigOptions) -> Void)?
)

CustomerIO.initialize(writeKey: writeKey, logLevel: .debug) { config in
    config.flushInterval = 30
    config.flushAt = 10
}
```

For `autoTrackScreenViews`, they should go with tracking module.